### DISCUSSION TITLE
non-targeting gRNAs

### DISCUSSION THREAD

**Discussion by nhu-github** [Katsevich-Lab/sceptre#152] [Q&A] [answered]
How to make the pipeline run if I have only one non-targeting gRNAs present?

See error message below: ----
Command error:
  Note: If you are on a Mac laptop or desktop, consider setting `parallel = TRUE` to improve speed. Otherwise, keep `parallel = FALSE`.

  Error in check_calibration_check_inputs(sceptre_object, n_calibration_pairs,  :
    Two or more non-targeting gRNAs must be present to run the calibration check when using the NT cells as the control group. gRNAs that are non-targeting should be assigned a gRNA group label of 'non-targeting' in the `grna_target_data_frame`.
  Calls: do.call -> <Anonymous> -> check_calibration_check_inputs
  Execution halted

**Comment by timothy-barry** [comment, marked as answer]
Hey Nan,

Are you running a low-MOI analysis in which you are using the NT cells as your control group? In this case, you will not be able to run a calibration check if you have only a single NT gRNA. My recommendation would be to skip the calibration check and just go straight to the power check and discovery analysis. In general we would recommend using at least ~10 NT gRNAs, if possible.
